scripts/upload_image.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+"""
+Upload an image to Google Cloud.
+
+Requires the following packages:
+  pip install google-cloud-storage # NOLINT
+
+Before running, you have to authenticate via the Google Cloud CLI:
+- Install it (https://cloud.google.com/storage/docs/gsutil_install)
+- Set up credentials (https://cloud.google.com/storage/docs/gsutil_install#authenticate)
+"""
+
+import argparse
+import hashlib
+import mimetypes
+
+from google.cloud import storage
+
+
+def content_hash(path: str) -> str:
+    h = hashlib.sha1()
+    b = bytearray(128 * 1024)
+    mv = memoryview(b)
+    with open(path, "rb", buffering=0) as f:
+        while n := f.readinto(mv):
+            h.update(mv[:n])
+    return h.hexdigest()
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Upload an image.")
+    parser.add_argument("path", type=str, help="Path to the image.")
+    args = parser.parse_args()
+
+    hash = content_hash(args.path)
+    object_name = f"{hash}_{args.path}"
```

Comment:
Looks like if I do `scripts/upload_image.py /Users/emilk/image.pg`, that will result in a really bad object_name. I think we should just use the filename here, not the whole path